admin/templates.md

```diff
@@ -18,3 +18,10 @@ Once you've filled out the form, Coder generates a custom Markdown snippet,
 which you can then add to your repository's `README.md`.
 
 ![Open In Coder Button](../assets/wac-badge.png)
+
+## Disabling workspaces as code
+
+In order to disable the workspaces as code feature, go to the **Admin >
+Templates** page and set _Enable using Workspace Templates_ to _OFF_.
```

Comment:
We may want to flip this to "Enabling workspaces as code". WAC is disabled by default.

Reply:
Awesome, didn't realize!